### DISCUSSION TITLE
Bubble-Card - picture as icon

### DISCUSSION THREAD

**Discussion by Nexow91** [Clooos/Bubble-Card#585] [Feature requests]
Picture instead of icon.

Hello first of all many thanks for the beautiful love-ace I am desperately trying to use a picture for the main button that is under "/local/". 
For the subbuttons there is the wiki (readme in the repo) which works but I have all variables like ".bubble-button" unfortunately this does not work. 

Is it even possible to use an image instead of an icon for the main button? 

I have also tried it with these variables without success. 

show_entity_picture: true
entity_picture: /local/icons8-autodesk-24.png

Is there any other way to use .png instead of icons (mdi:switch) ?

**Comment by vhsdream** [comment]
Bringing this FR back from the dead - I think this would be a cool feature. My use case is that I am using the heading in a pop-up as a link to an external site. I'd like to use the logo of the service in place of the icon. I tried using 'button-card'-style javascript like so:

`[[[ return '<img src="/local/icons/vikunja.png>' ]]]`

But unfortunately that did not work. Does anyone out there have any further tips for achieving this?

**Comment by rononline** [comment]
```
.bubble-entity-picture{
        background-image: url('/local/icons/vikunja.png') !important;
        display: block !important;    
        background-position: center center;
        background-size: 75%;
        background-repeat: no-repeat;
        overflow: hidden;
        }
 ```

**Comment by vhsdream** [comment]
Thank you, 

But I ended up creating a template image helper and then using that as the entity, which ended up showing the icon. 

On Tue, Jan 28, 2025, at 1:56 PM, Ron wrote:
> 
> 
> `.bubble-entity-picture{
>         background-image: url('/local/icons/vikunja.png') !important;
>         display: block !important;    
>         background-position: center center;
>         background-size: 75%;
>         background-repeat: no-repeat;
>         overflow: hidden;
>         }
`

**Comment by joshrmiller** [comment]
How would you replace the icon on a sub button with a new mdi:icon?  If you're using an existing mdi in HA, what path do you use for background-image?
